src/main/java/org/takes/rq/RqLive.java

```diff
@@ -45,7 +45,7 @@
  * @since 0.1
  */
 @EqualsAndHashCode(callSuper = true)
-public final class RqLive extends RqWrap {
+public class RqLive extends RqWrap {
```

Comment:
@essobedo check this: http://www.yegor256.com/2014/11/20/seven-virtues-of-good-object.html#7-his-class-is-either-final-or-abstract